index.html

```diff
@@ -14,10 +14,12 @@
                 </div>
                 <section class="navigation">
                     <ul>
+                        <a href="#">
                         <li>Home</li>
```

Comment:
Don't make them one link, You should have created separate links for all li elements